django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -30,7 +31,7 @@ async def receive(self, text_data):
             {"role": message.role, "text": message.text} for message in session_messages
         ]
         url = (
-            URL.build(scheme=settings.WEBSOCKET_SCHEME, host=settings.CORE_API_HOST, port=settings.CORE_API_PORT)
+            URL.build(scheme="ws", host=settings.CORE_API_HOST, port=settings.CORE_API_PORT)
```

Comment:
this needs to be hard coded as it is the connection to core-api within the same network so should not be `wss`